### DISCUSSION TITLE
File upload abort option

### DISCUSSION THREAD

**Discussion by mozex** [livewire/livewire#4626] [3. Feature ideas]
Hi, it would be awesome if there was an option to cancel the uploading file, when a big file is being uploaded,  we may want to abort it and upload another file before the upload finishes.
Thanks...

**Comment by joshhanley** [comment]
@mozex there was a PR #3216 for this, but the PR was incomplete. If someone can submit a complete PR which addresses Caleb's concerns in that PR, that'd be great!

Also could you share some example code of how you would use it?
